### PR TITLE
Fix transforms streamlit

### PR DESCRIPTION
**Pull request by linus-md**
This PR requested by @CYHSM  adds `transforms=['Affine', 'Affine', 'SyNAggro']` to the streamlit preprocessing. Thereby making more aggressive alignment the default. Which should improve alignment significantly.

### CODE REVIEW

File: .DS_Store
Comment (Remi-Gau) [inline .DS_Store:1]: Can you remove this file?

Comment (Remi-Gau) [reply .DS_Store:1]: probably best to also update the gitignore to avoid having commits with this type "pollution" coming from MasOS.

**Comment by linus-md** [comment]
Hope it is to your liking now.

**Comment by Remi-Gau** [comment]
merging as the failures of CI are unrelated (see #40)

**Pull request merged**